hasura.planx.uk/migrations/1747216207505_alter_table_public_templated_flow_edits_add_unique_flow_id/up.sql

```diff
@@ -0,0 +1,4 @@
+alter table "public"."templated_flow_edits" drop constraint "templated_flow_edits_node_id_flow_id_key";
+alter table "public"."templated_flow_edits" add constraint "templated_flow_edits_flow_id_key" unique ("flow_id");
+alter table "public"."templated_flow_edits" drop column "node_id" cascade;
+alter table "public"."templated_flow_edits" rename column "node_data" to "data";
```

Comment:
Flip-flopped on the table structure again! Dropped column `node_id` so table is now unique based on `flow_id` alone and we just record a single `data` jsonb (this has benefit of capturing changes to Node `edges` as well as `data`!)